js/screens/Journal/Journal.js

```diff
@@ -1,12 +1,12 @@
-import React from "react";
+import React, {Fragment} from "react";
 import PropTypes from "prop-types";
-import { Text, View } from "react-native";
+import Loading from "../../components/Loading";
 
 const Journal = () => {
   return (
-    <View>
-      <Text>Journal</Text>
-    </View>
+    <Fragment>
```

Comment:
remove Fragment Wrapper. there are not more than 2 component in there. Just return your Loading component.

Reply:
done